src/Razor/test/Microsoft.VisualStudio.Razor.Integration.Test/InProcess/InProcComponent.cs

```diff
@@ -47,6 +47,14 @@ protected async Task<TService> GetComponentModelServiceAsync<TService>(Cancellat
             return componentModel.GetService<TService>();
         }
 
+        protected async Task ExecuteCommandAsync(string commandName, CancellationToken cancellationToken, string args = "")
+        {
+            await JoinableTaskFactory.SwitchToMainThreadAsync();
+
+            var dte = await GetRequiredGlobalServiceAsync<SDTE, EnvDTE.DTE>(cancellationToken);
+            dte.ExecuteCommand(commandName, args);
```

Comment:
DTE should be avoided where possible. In this case, the replacement is:

```suggestion
            var dispatcher = await GetRequiredGlobalServiceAsync<SUIHostCommandDispatcher, IOleCommandTarget>(cancellationToken);
            ErrorHandler.ThrowOnFailure(dispatcher.Exec(...));
```

Reply:
Looks like this was grabbed from `IntegrationTests` instead of `New.IntegrationTests`. Regardless, it's not actually used so the easy fix is just to remove it.